definitions/procedures/restore/configs.rb

```diff
@@ -45,5 +46,10 @@ def clean_conflicting_data
       # tar is unable to --overwrite dir with symlink
       execute('rm -rf /usr/share/foreman-proxy/.ssh')
     end
+
+    def reset_qpid_jrnls
+      # on restore without pulp data qpid fails to start
```

Comment:
might be worth adding `https://access.redhat.com/solutions/4645231` to this comment as reference.